Master adcs v1.2 with eds utils master

dsigma opened the pull request.

ryanpdx reviewed (commented):
Remove OD.c and OD.h from git. These should be ignored by git.

Why is this a new app? Shouldn't this be app_imu?

Update `.github/workflows/tests.yaml` to include this new app

dsigma commented:
I've removed the OD files. I've been working on app_adcs for a long time. I'm not sure what the history is on app naming. I've modified the tests.yml file to included app_adcs and pushed the changes to git.
